src/main/offline/io/stargate/sgv2/jsonapi/service/cqldriver/sstablewriter/OfflineWriterSessionStatus.java

```diff
@@ -0,0 +1,11 @@
+package io.stargate.sgv2.jsonapi.service.cqldriver.sstablewriter;
+
+public record OfflineWriterSessionStatus(
+    String sessionId,
+    String keyspace,
+    String tableName,
+    String ssTableOutputDirectory,
+    int fileWriterBufferSizeInMB,
+    int dataDirectorySizeInBytes,
```

Comment:
Should this be `long` instead? (max of `int` is about 2 gigs)

Reply:
Good catch. Fixed in this commit [ae61b15](https://github.com/stargate/data-api/pull/905/commits/ae61b15f85a0e05c1273572db5e6b8ee6bab0163).